Assets/BossRoom/Scripts/Client/Game/Action/ActionVisualization.cs

```diff
@@ -66,6 +66,14 @@ public void OnStoppedChargingUp(float finalChargeUpPercentage)
             }
         }
 
+        public void OnTeleport()
```

Comment:
Adding one single action has a lot of rippling effect. Why is the generic ActionVisualization aware of teleportation? Same for the other changes in this PR, why does it change so many files?

Reply:
(see earlier comment)

removed `OnTeleport()` from `ActionVisualization`, `ActionFX`, and `DashAttackActionFX`